pkg/codegen/java/gen.go

```diff
@@ -1754,7 +1754,7 @@ func (mod *modContext) gen(fs fs) error {
 			}
 			return javaUtilitiesTemplate.Execute(ctx.writer, javaUtilitiesTemplateContext{
 				Name:        pkgName.String(),
-				PackagePath: strings.ReplaceAll(mod.packageName, ".", "/"),
```

Comment:
do not use Java package name, because Bootstrap logic expects the Pulumi package in the path